modular_skyrat/modules/verbs/code/modules/client/verbs/looc.dm

```diff
@@ -40,22 +40,13 @@
 
 	var/list/heard = get_hearers_in_view(7, get_top_level_mob(src.mob))
 
-	//SKYRAT EDIT ADDITION BEGIN - AI QoL
-	var/ai_heard_it = FALSE
 	//so the ai can post looc text
 	if(istype(mob,/mob/living/silicon/ai))
```

Comment:
```suggestion
	if(istype(mob,/mob/living/silicon/ai))
	    var/mob/living/silicon/ai/ai = mob
```
you marked this as resolved but didn't actually do anything and is still throwing errors